ktor-http/common/src/io/ktor/http/auth/HttpAuthHeader.kt

```diff
@@ -287,6 +290,14 @@ public sealed class HttpAuthHeader(public val authScheme: String) {
             }
         )
 
+        /**
+         * Generates an [AuthScheme.Bearer] challenge as a [HttpAuthHeader].
+         */
+        public fun bearerAuthChallenge(scheme: String, realm: String?): HttpAuthHeader = Parameterized(
```

Comment:
realm can have `null` value by default

Reply:
don't forget to run `./gradlew apiDump` after you add it

Reply:
Sure.  All done